godot-core/src/builtin/meta/class_name.rs

```diff
@@ -36,6 +36,12 @@ impl ClassName {
     }
 }
 
+impl From<ClassName> for StringName {
+    fn from(class_name: ClassName) -> Self {
+        class_name.backing
+    }
+}
+
```

Comment:
Maybe an explicit `string_name()` method is clearer? I'm having a bit a hard time deciphering `.into()` conversions all over the code, and this internal API doesn't need to be ergonomic 😉

Reply:
Haha, I did that at first and found it unrusty :) I updated the (only) call site to explicitly call `StringName::from()`.

Reply:
Still not a huge fan, but there's more important stuff to argue about 😛